### ISSUE TITLE
Database for problematic games

### ISSUE THREAD

**Issue by gameblabla** [networkfusion/alt64#32]
It seems that altera64 uses the header to determine which CIC chip it should use.
However in some cases, that may not be enough to run a game properly.
A few examples :
- Pokemon Stadium 2 (France) will refuse to run unless it is forced to PAL for the country & TV setting.
- Alek64 games requires the 5101 CIC but it's not always being detected properly.
- Similar issue for DD64 converted carts : they require the 5167 CIC but it's not being detected to use it.

As for the database implementation itself, we could either use a hardcoded (a bunch of arrays) one, or an ini/yaml file.

**Comment by networkfusion** [comment]
strange aleck64 and 64DD games tested ok when I implemented them... But I am using an ED64V3 with ultraCIC… so perhaps it is to do with that?!

**Comment by networkfusion** [comment]
also FYI, this may be of use (also me) http://krikzz.com/forum/index.php?topic=7991.msg61568#msg61568

**Comment by gameblabla** [comment]
> strange aleck64 and 64DD games tested ok when I implemented them... But I am using an ED64V3 with ultraCIC… so perhaps it is to do with that?!

Yeah that could be possible. (i tested it against an ED64P btw but that is mostly a hardware clone of the Everdrive v1/v2 without the UltraCIC chip). If you take a random Alek64 game and press C-right on one, you may notice that it is not being set to the proper CIC by default. (at least it does for me)

> also FYI, this may be of use (also me) http://krikzz.com/forum/index.php?topic=7991.msg61568#msg61568

I looked at it and all i could find is talk about the save issue with Pokemon stadium 2 due to the overlapping with the EEPROM and rom i believe ?

**Comment by networkfusion** [comment]
Yeah the point of my mod was to make it compatible with the ED64 V3 (the only device I own), so may be missing stuff around older hardware and cannot test against it... I guess the same goes for all OS's with regards to pokemon stadium... the older devices just don't have enough save memory for that game... the V3 has dedicated ram for it....
